runtime/src/bank/pyth_accumulator.rs

```diff
@@ -367,3 +382,32 @@ pub fn update_v2(bank: &Bank) -> std::result::Result<(), AccumulatorUpdateErrorV
 
     update_v1(bank, &v2_messages, any_v1_aggregations)
 }
+
+pub fn compute_publisher_stake_caps(
```

Comment:
it's cleaner if you call this one if the feature is active and it returns a Message instead of mutating the vector.

Reply:
fixed